### PR TITLE
chore: add contextual docs edits

### LINKED ISSUE #5121: Contextual docs should follow our docs standards
## Details

Our docs avoid first person language

Our docs use a few words as possible to get the point across.

Our docs embody the "wise friend" from our [tone guide](https://benlikes.us/tone)

## Additional Context

I'm going to submit a PR with changes. 

## Acceptance Criteria

-   [x] First Person language is removed
-   [x] Typos are eliminated
-   [x] Confusing language is removed.

### PR DESCRIPTION
**Pull request by Benunc**
Resolves #5121 

## Description

Made a few contextual docs changes.

## Affects

Only affects the onboarding pages. (requires `npm run dev` or similar to actually see the changes)

## Pre-review Checklist

-   [x] Acceptance criteria satisfied and marked in related issue
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

## Testing Instructions

run `npm` to make sure that the changes are compiled to be visible.

### CODE REVIEW

File: src/Onboarding/Setup/templates/index.html.php [hunk 1]
```diff
@@ -58,7 +58,7 @@
 												: $this->image( 'paypal@2x.min.png' ),
 							'icon_alt'    => esc_html__( 'PayPal', 'give' ),
 							'title'       => esc_html__( 'Connect to PayPal', 'give' ),
-							'description' => esc_html__( 'PayPal is synonymous with nonprofits and online charitable gifts. It’s been the go-to payment merchant for many of the world\'s top NGOs. Accept PayPal, Credit and Debit Cards, and more using PayPal’s Smart Buttons without any added platform fees.', 'give' ),
+							'description' => esc_html__( 'PayPal is synonymous with nonprofits and online charitable gifts. It’s been the go-to payment merchant for many of the world\'s top NGOs. Allow donors to log in to their PayPal account to complete the donation.', 'give' ),
```
Comment (DevinWalker) [inline src/Onboarding/Setup/templates/index.html.php:61]: @Benunc not sure I like this edit. They don't actually have to log into their PayPal account if they don't have one or want to.

Comment (Benunc) [reply src/Onboarding/Setup/templates/index.html.php:61]: @DevinWalker is that true in 2.8.0, though? as of 2.9.0 the removed line will be true. right now, it's still just PayPal Standard.

Comment (DevinWalker) [reply src/Onboarding/Setup/templates/index.html.php:61]: It is true in 2.8.0 as well @Benunc

**Comment by kevinwhoffman** [comment]
@Benunc Please tag me when you resolve the pending comments and this is ready for review.

File: src/Onboarding/Setup/templates/index.html.php (hunk 1 above)
Comment (Benunc) [reply src/Onboarding/Setup/templates/index.html.php:61]: @DevinWalker credit and debit cards are only possible for a small percentage of users, in my experience. Smart buttons are not at all possible. I much prefer the second edit for setting expectations that this gateway currently redirects to PayPal for the donation to be complete.

Comment (kevinwhoffman) [reply src/Onboarding/Setup/templates/index.html.php:61]: @DevinWalker Does this work? I kept the mention of credit and debit cards, but removed the mention of smart buttons which to my knowledge won't be available until 2.9.0:
> PayPal is synonymous with nonprofits and online charitable gifts. It’s been the go-to payment merchant for many of the world\'s top NGOs. Accept PayPal, Credit and Debit Cards without any added platform fees.

Comment (kevinwhoffman) [reply src/Onboarding/Setup/templates/index.html.php]: Resolving this as it was approved by Devin in Slack.

**Comment by kevinwhoffman** [comment]
@kjohnson Can you resolve this merge conflict? I don't want to undo anything you did with regards to the Stripe connect button.

**Comment by kjohnson** [comment]
@kevinwhoffman I resolved the conflicts. I'm pulling it down to test and double check.

**Comment by kjohnson** [comment]
@kevinwhoffman ready to go! Back to you.

**Pull request merged**